drivers/bridge/setup_bridgenetfiltering.go

```diff
@@ -64,11 +64,12 @@ func checkBridgeNetFiltering(config *networkConfiguration, i *bridgeInterface) e
 		if err != nil {
 			logrus.Warnf("failed to check %s forwarding: %v", ipVerName, err)
 		} else if enabled {
-			enabled, err := getKernelBoolParam(getBridgeNFKernelParam(ipVer))
+			bridgeName := i.Link.Attrs().Name
```

Comment:
I see that we already have something similar in https://github.com/docker/libnetwork/blob/feeff4f0a3fd2a2bb19cf67c826082c66ffaaed9/drivers/bridge/setup_bridgenetfiltering.go#L55
Can we use any one way of deriving the `bridgeName` and reference it everywhere else

Reply:
Done.  It got mixed up with the requested signoff and commit comment changes so has ended up in the same commit, apologies.